### ISSUE TITLE
Python 3.6 problems?

### ISSUE THREAD

**Issue by CesMak** [jgrassler/mkdocs-pandoc#28]
Hey there, just tried your tool and got the following error:

````
mkdocs2pandoc > mydocs.pd
Traceback (most recent call last):
  File "/home/markus/anaconda3/bin/mkdocs2pandoc", line 11, in <module>
    sys.exit(main())
  File "/home/markus/anaconda3/lib/python3.6/site-packages/mkdocs_pandoc/cli/mkdocs2pandoc.py", line 80, in main
    for line in pconv.convert():
  File "/home/markus/anaconda3/lib/python3.6/site-packages/mkdocs_pandoc/pandoc_converter.py", line 165, in convert
    lines = mkdocs_pandoc.filters.tables.TableFilter().run(lines)
  File "/home/markus/anaconda3/lib/python3.6/site-packages/mkdocs_pandoc/filters/tables.py", line 177, in run
    ret.extend(self.convert_table(block))
  File "/home/markus/anaconda3/lib/python3.6/site-packages/mkdocs_pandoc/filters/tables.py", line 91, in convert_table
    for i in range(0, len(self._split_row(lines_orig[0], has_border))):
TypeError: _split_row() takes 2 positional arguments but 3 were given

```

anyone else had this error before? 

Best - Markus

**Comment by HedgehogCode** [comment]
See #22

**Comment by charlesreid1** [comment]
TL; DR: breaks because of changes in the markdown library. downgrade:

```
pip install --upgrade markdown==2.6.7
```

**Comment by harrypalheta** [comment]
> TL; DR: breaks because of changes in the markdown library. downgrade:
> 
> ```
> pip install --upgrade markdown==2.6.7
> ```

In my case it worked by putting the `pip3`.
